src/public_pool/tor_public_pool.sh

```diff
@@ -1,34 +1,39 @@
 function enable_tor_public_pool {
-if [[ $OS == Mac ]] ; then no_mac ; return 1 ; fi
+    if [[ $OS == Mac ]]; then no_mac; return 1; fi
```

Comment:
I don't mind much either way, but I use spaces before every ; all through Parmanode.
If I'm going to use C or C++, I don't use the space.